src/pages/guides/review/index.md

```diff
@@ -6,17 +6,39 @@ title: Submission and Review
 description: This page explains the submission and review process.
 contributors:
   - https://github.com/amandahuarng
+  - https://github.com/nimithajalal
 ---
 
-# Submission and Review
+## Overview
 
-Adobe reviews all applications that integrate Adobe Express Embed SDK before enabling it for public use.
+This guideline outlines the process for reviewing and submitting an integration using the Adobe Express Embed SDK. Whether you’re embedding the Full Editor, Quick Actions, or Modules, adhering to these guidelines will ensure a seamless experience for users and maintain consistency across all implementations.
 
-## Before you submit
+<InlineAlert slots="text" />
+
+We aim to review your integration within 10 business days of submission, and will let you know if it is accepted, or if any changes need to be made.
```

Comment:
I'd use bold if we are confident that the review team can make it in 10 days :-) 

We aim to review your integration **within 10 business days of submission**, and will let you know if it is accepted, or if any changes need to be made.

Reply:
We are not really confident here.